samples/matter/thermostat/README.rst

```diff
@@ -0,0 +1,312 @@
+You must pair the device with the Matter controller over Bluetooth® LE to get the configuration from the controller to use the device within a Thread or a Wi-Fi network.
+You can enable the controller after :ref:`building and running the sample <matter_thermostat_network_testing>`.
+
+To pair the device, the controller must get the :ref:`matter_thermostat_network_mode_onboarding` from the Matter accessory device and commission the device into the network.
+
+
+Commissioning in Matter
+-----------------------
+
+In Matter, the commissioning procedure takes place over Bluetooth LE between a Matter accessory device and the Matter controller, where the controller has the commissioner role.
+When the procedure has completed, the device is equipped with all information needed to securely operate in the Matter network.
+
+During the last part of the commissioning procedure (the provisioning operation), the Matter controller sends the Thread or Wi-Fi network credentials to the Matter accessory device.
+As a result, the device can join the IPv6 network and communicate with other devices in the network.
+
+.. _matter_thermostat_network_mode_onboarding:
+
+Onboarding information
+++++++++++++++++++++++
+
+When you start the commissioning procedure, the controller must get the onboarding information from the Matter accessory device.
+The onboarding information representation depends on your commissioner setup.
+
+For this sample, you can use one of the following :ref:`onboarding information formats <ug_matter_network_topologies_commissioning_onboarding_formats>` to provide the commissioner with the data payload that includes the device discriminator and the setup PIN code:
+
+  .. list-table:: Thermostat sample onboarding information
+     :header-rows: 1
+
+     * - QR Code
+       - QR Code Payload
+       - Manual pairing code
+     * - Scan the following QR code with the app for your ecosystem:
+
+         .. figure:: ../../../doc/nrf/images/matter_qr_code_thermostat.png
+            :width: 200px
+            :alt: QR code for commissioning the thermostat device
+
+       - MT:O4CT3KQS02ZZ
+       - 548G00
+
+Configuration
+*************
+
+|config|
+
+
+
+Access Control List
+===================
+
+The Access Control List (ACL) is a list related to the Access Control cluster.
+The list contains rules for managing and enforcing access control for a node's endpoints and their associated cluster instances.
+In this sample's case, this allows the temperature measurement devices to receive messages from the thermostat and to get temperature data.
+
+You can read more about ACLs on the :doc:`matter:access-control-guide` in the Matter documentation.
+
+.. _matter_thermostat_sample_binding:
+
+Binding
+=======
+.. include:: ../light_switch/README.rst
+   :start-after: matter_light_switch_sample_binding_start
+   :end-before: matter_light_switch_sample_binding_end
+
+This sample can operate in two modes:
```

Comment:
This piece is wrongly formatted:
https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-11737/nrf/samples/matter/thermostat/README.html#binding
Please use bullet points.